content/docs/guides/javascript/js-api.md

```diff
@@ -116,6 +116,19 @@ The following methods and configuration options accept a function taking a `Cont
 - `preOps()`
 - `postOps()`
 
+## Referencing includes files
+
+Just like in `.sqlx` files, you can reference any `includes` function, macro or constant inside a `.js` file. However, the syntax differs slightly.
+
+For example, if you would like to reference `SERIVCE_NAME` and `SERVICE_ID` from the `includes/service` file, you must declare these constants using the JavaScript builtin `require` function.
```

Comment:
The rules here are a little tricky:
- all top-level `includes` .js files are automatically imported, everywhere (i.e. if you have a top-level `includes.constants.js` file, a `constants` object is available for use in all files, whether .sqlx or .js, without any need for a `require`)
- all other `includes` .js files are not automatically imported, and wherever they are used, whether in .sqlx or .js files, they must be explicitly imported with `require()`

So it's more about the location of the file in `includes` than where you're using things exported by that file - does that make sense?

Reply:
Gotcha. Updated.